pkg/beholder/utils/metadata.go

```diff
@@ -0,0 +1,98 @@
+package utils
+
+import (
+	"encoding/hex"
+
+	"go.opentelemetry.io/otel/attribute"
+)
+
+const (
+	// WorkflowExecutionIDShortLen is the length of the short version of the WorkflowExecutionId (label)
+	WorkflowExecutionIDShortLen = 3 // first 3 characters, 16^3 = 4.096 possibilities (mid-high cardinality)
+)
+
+// TODO: Refactor as a proto referenced from the other proto files (telemetry messages)
```

Comment:
For now or later? If later, is there a ticket to link?

Reply:
This TODO was carried over from the Aptos repo, and thus I think is out of scope for this PR. cc: @krebernisak if you have more context